Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:14-alpine as build
+
+ENV NODE_ENV=production
+
+WORKDIR /build-dir
+
+COPY package*json ./
+
+RUN npm ci
+
+COPY . .
+
+RUN npm run build
+
+RUN echo "commit: $COMMIT_SHA" >> ./commit.sha
+
+########################################################################################################################
+
+FROM node:14-alpine
+
+ENV NODE_ENV=production
```

Comment:
can you add the required env variables please? 

`ENV NODE_ENV=production VIDEO_API_API_KEY=$VIDEO_API_API_KEY VIDEO_API_API_SECRET=$VIDEO_API_API_SECRET`